scripts/docker-entrypoint.sh

```diff
@@ -40,6 +40,11 @@ _sed-in-place() {
 	rm "$tempFile"
 }
 
+_metrics_collector_supported() {
+	# currently, metrics collector does not work on arm64
+	[ "$(uname -m)" != "aarch64" ]
+}
```

Comment:
I don't know the best way to handle this. Somehow we need to disable the metrics collector for arm64. This appears to be working, but I'm open to suggestions.